### PR TITLE
Adds cosmic-client and cosmic-ui submodules

### PR DESCRIPTION
**Pull request by wilderrodrigues**
This PR comprises the changes for UI, Client and Build.

Concerning UI and Client, they are now under:
- cosmic/comic-client
- cosmic/comic-ui

In order to be able to run jetty form the cosmic directory, I also moved the build directory from cosmic-core.

### CODE REVIEW

**Comment by wilderrodrigues** [comment]
Ping @borisroman @bheuvel @michaelandersen @remibergsma @miguelaferreira 

Please, keep an eye on this one. Currently under tests.

File: pom.xml [hunk 1]
```diff
@@ -103,6 +103,7 @@
     <module>cosmic-plugin-hypervisor-ovm3</module>
     <module>cosmic-plugin-hypervisor-kvm</module>
     <module>cosmic-plugin-hypervisor-xenserver</module>
+    <module>cosmic-client</module>
```
Comment (borisroman) [inline pom.xml]: Kind of nit-picking; but do want this in alphabetical order?

File: .gitmodules [hunk 2]
```diff
@@ -22,3 +22,11 @@
 	path = cosmic-packaging
 	url = git@github.com:MissionCriticalCloud/cosmic-packaging.git
 	branch = master
+[submodule "cosmic-ui"]
```
Comment (borisroman) [inline .gitmodules:29]: Same here..., alphabetical maybe?

File: pom.xml (hunk 1 above)
Comment (wilderrodrigues) [inline pom.xml]: nit-picking indeed, but with a good point. ;)

once the tests are done, I will update the files and push again, so you can merge all PRs.

Cheers,
Wilder

**Comment by wilderrodrigues** [comment]
Ping @borisroman @bheuvel @michaelandersen @remibergsma @miguelaferreira 

Tests done! I also did some click around on the UI in order to check if everything is working fine.

The submodules and maven modules are now in alphabetical order.

```
[root@cs2 integration]# less /tmp//MarvinLogs/test_vm_life_cycle_PIP88K/results.txt 
Test router internal advanced zone ... === TestName: test_02_router_internal_adv | Status : SUCCESS ===
ok
Test restart network ... === TestName: test_03_restart_network_cleanup | Status : SUCCESS ===
ok
Test router basic setup ... === TestName: test_05_router_basic | Status : SUCCESS ===
ok
Test router advanced setup ... === TestName: test_06_router_advanced | Status : SUCCESS ===
ok
Test stop router ... === TestName: test_07_stop_router | Status : SUCCESS ===
ok
Test start router ... === TestName: test_08_start_router | Status : SUCCESS ===
ok
Test reboot router ... === TestName: test_09_reboot_router | Status : SUCCESS ===
ok
Test reset virtual machine on reboot ... === TestName: test_01_reset_vm_on_reboot | Status : SUCCESS ===
ok
Test advanced zone virtual router ... === TestName: test_advZoneVirtualRouter | Status : SUCCESS ===
ok
Test Deploy Virtual Machine ... === TestName: test_deploy_vm | Status : SUCCESS ===
ok
Test Multiple Deploy Virtual Machine ... === TestName: test_deploy_vm_multiple | Status : SUCCESS ===
ok
Test Stop Virtual Machine ... === TestName: test_01_stop_vm | Status : SUCCESS ===
ok
Test Start Virtual Machine ... === TestName: test_02_start_vm | Status : SUCCESS ===
ok
Test Reboot Virtual Machine ... === TestName: test_03_reboot_vm | Status : SUCCESS ===
ok
Test destroy Virtual Machine ... === TestName: test_06_destroy_vm | Status : SUCCESS ===
ok
Test recover Virtual Machine ... === TestName: test_07_restore_vm | Status : SUCCESS ===
ok
Test migrate VM ... SKIP: At least two hosts should be present in the zone for migration
Test destroy(expunge) Virtual Machine ... === TestName: test_09_expunge_vm | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 18 tests in 3940.785s

OK (SKIP=1)
/tmp//MarvinLogs/test_vm_life_cycle_PIP88K/results.txt (END)
```

**Pull request merged**